tests/browser/ckeditor.jsx

```diff
@@ -278,6 +276,54 @@ describe( 'CKEditor Component', () => {
 				} );
 			} );
 		} );
+
+		it( 'uses the freshest data - component mounted, editor not loaded', () => {
+			const initialData = '<p>Initial data</p>';
+			const changedData = '<p>Changed data</p>';
+			const component = createEditor( {
+				data: initialData
+			} );
+
+			// Component is already mounted, editor is not loaded, `data` prop changes
+			component.setProps( {
+				data: changedData
+			} );
+
+			return waitForEditor( component, 'loaded' ).then( editor => {
+				sandbox.spy( editor, 'setData' );
+
+				return waitForEditor( component );
+			} ).then( editor => {
+				expect( editor.getData().trim() ).to.equal( changedData );
+			} );
+		} );
+
+		it( 'uses the freshest data on mount - component mounted, editor loaded', () => {
```

Comment:
Please, add tickets references
```
// (#114, #127)
```
Like we have here: https://github.com/ckeditor/ckeditor4-react/blob/04938080004fc385e232da53fdf4e130dda70971/tests/browser/ckeditor.jsx#L69-L70

Reply:
Done!